### ISSUE TITLE
Cannot prevent a subdomain from being unonionized

### ISSUE THREAD

**Issue by micahflee** [alecmuffett/eotk#65]
My config file has:

```
hardmap %NEW_V3_ONION% theintercept.com
```

And also:

```
set redirect_host ^join\\..*\\.onion$,307,https://join.theintercept.com
```

This should redirect `https://join.[xxx].onion` to `https://join.theintercept.com`. And it appears that it does, except EOTK also see the `theintercept.com` and rewrites it again to `https://join.[xxx].onion`, so it ends up in an infinite redirect loop.

**Comment by alecmuffett** [comment]
Revised and apparently working fix in #66, @micahflee

**Comment by micahflee** [comment]
I confirmed that your fix works!
